charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```
@@ -67,7 +67,11 @@ data:
       webhook_endpoint: "{{ .Values.config.notifications.webhook_endpoint }}"
     paymail:
       beef:
-        block_header_service_url: 'http://{{- printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" }}/api/v1/chain/merkleroot/verify'
+        {{- if .Values.config.paymail.beef.block_headers_service_url }}
+        block_headers_service_url: {{ .Values.config.paymail.beef.block_headers_service_url }}
+        {{- else }}
+        block_headers_service_url: '{{ default (printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-") }}/api/v1/chain/merkleroot/verify'
```

You removed http prefix

```suggestion
        block_headers_service_url: 'http://{{- printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" }}/api/v1/chain/merkleroot/verify'
```